jazz_test-lambda/index.js

```diff
@@ -0,0 +1,107 @@
+// =========================================================================
+// Copyright ©  2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+Nodejs Template Project
+@author:
+@version: 1.0
+ **/
+'use strict';
+const errorHandlerModule = require("./components/error-handler.js"); //Import the error codes module.
+const responseObj = require("./components/response.js"); //Import the response module.
+const configObj = require("./components/config.js"); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const validateARN = require("./components/validate-arn.js");
+const validateJSON = require("./components/validate-json.js");
+const aws = require('aws-sdk');
+var handler = (event, context, cb) => {
+
+  //Initializations
+  var errorHandler = errorHandlerModule();
+  var config = configObj(event);
+  logger.init(event, context);
+  var AWS_REGION = config.AWS_REGION;
+  try {
+    var testResponse = {
+      "StatusCode": 200,
+      "execStatus": 0
+    };
+    if (event && event.method && event.method === 'POST') {

```

Comment:
what happens if this is false? As per your code, you are going to skip calling callback which makes this function fail. You have to handle this this gracefully. 

Please handle the else scenarios (you can look at other APIs for reference).